src/core/server/http_resources/integration_tests/http_resources_service.test.ts

```diff
@@ -65,7 +65,6 @@ describe('http resources service', () => {
         resources.register({ path: '/render-core', validate: false }, (context, req, res) =>
           res.renderAnonymousCoreApp({
             headers: {
-              'content-security-policy': "script-src 'unsafe-eval'",
               'x-kibana': '42',
```

Comment:
Same as other comment above

```suggestion
              'content-security-policy': "script-src 'unsafe-eval'",
              'x-kibana': '42',
```